WORKSPACE

```diff
@@ -1,84 +1,64 @@
-workspace(name = "io_bazel_rules_play_routes")
+workspace(name = "rules_play_routes")
```

Comment:
Any particular reason for the name change?

Reply:
I recall seeing some guidance that names under `io_bazel_` were for official bazel organization things. I can't find it anymore though. This should probably be something like `com_github_lucidsofware_XXX`. 

There is also some guidance here (https://docs.bazel.build/versions/master/be/functions.html#workspace,
https://docs.bazel.build/versions/master/skylark/lib/globals.html#workspace) about names being reverse DNS style but they also don't mention the `io_bazel_` thing.

Reply:
> I recall seeing some guidance that names under io_bazel_ were for official bazel organization things. 

I'd suspect this too. I reckon the name change is an improvement.